app/tree/gui/src/ts/twxplore/gui/geo/api/queries/ResultsQuery.graphql

```diff
@@ -0,0 +1,91 @@
+query ResultsQuery($selectionInput: SelectionInputFieldsInput!) {
```

Comment:
Better naming, please. I name the query after the top-level component (e.g., the thing that has a Frame) that uses the query. Since only top-level components should be making queries (= one per page), this usually works.